index.html

```diff
@@ -289,6 +289,80 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <form class="contact-form">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+
+        <input type="text" placeholder="Full name" />
```

Comment:
Your Name Field should accept up to 30 characters of the name. Please consider adding the following `max-length `property to your Input Tag. Moreover, there needs a restriction so a user can't leave the Name Input empty. To achieve all the above you can use the following piece of code as an example.

`<input type="text" placeholder="Full name" maxlength="30" required/>`